src/foam/lib/json/Outputter.java

```diff
@@ -96,17 +97,33 @@ protected void outputNull() {
   }
 
   public void outputString(String s) {
-    writer_.append("\"");
-    writer_.append(escape(s));
-    writer_.append("\"");
+    if ( multiLineOutput_ && s.indexOf('\n') >= 0 ) {
+      writer_.append("\n  \"\"\"\n    ");
+      s = ltrim(s, 4);
+      writer_.append(escapeMultiline(s));
+      writer_.append("\n  \"\"\"");
+    }
+    else {
+      writer_.append("\"");
+      writer_.append(escape(s));
+      writer_.append("\"");
+    }
   }
 
   public String escape(String s) {
-    return s
-      .replace("\\", "\\\\")
-      .replace("\"", "\\\"")
-      .replace("\t", "\\t")
-      .replace("\n","\\n");
+    return s.replace("\\", "\\\\")
+            .replace("\"", "\\\"")
+            .replace("\t", "\\t")
+            .replace("\n","\\n");
+  }
+
+  public String escapeMultiline(String s) {
+    s = s.replace("\\", "\\\\")
+         .replace("\\\"", "\"")
+         .replace("\n","\\n")
+         .replace("\t", "\\t");
```

Comment:
don't need to escape tabs either I don't think.

Reply:
this was there before; I did not add this in so someone probably put it there for a reason

Reply:
This is the multiline escape.  What I'm saying is we don't need to escape tabs in multiline mode, we can just leave them as literal tabs.